client/src/utils/date.js

```diff
@@ -0,0 +1,22 @@
+export function getCurrentDate() {
+  const currentDate = new Date();
+  // Following code is super hacky, gonna be changed, just needed it to work for now.
+  // Need fixed format for currentDate to be able to read data from database.
```

Comment:
yes.

Reply:
@phlgr Any sort of advice on how to simplify this part of the code to still always get the data formed as "yyyy-mm-dd"?

Reply:
Nvm, has been solved.